Controller/LogController.php

```diff
@@ -10,9 +10,26 @@ class LogController extends Controller
 {
     public function createAction(Request $request)
     {
-        $level = $request->query->get('level');
-        $message = $request->query->get('msg');
-        $context = $request->query->get('context', array());
+        if("GET" == $request->getMethod()){
+            $level = (string) $request->query->get('level');
+            $message = (string) $request->query->get('msg');
+            $context = (array) $request->query->get('context', array());
+
+        } else {
+            $postData = json_decode($request->getContent());
+            $level = (string) $postData->level;
+            $message = (string) $postData->msg;
+            $context = (array) $postData->context;
+            $stacktrace = "";
+            foreach ($postData->stack as $stackframe) {
+                $fileName = property_exists($stackframe, 'fileName') ? $stackframe->fileName : "-";
+                $lineNumber = property_exists($stackframe, 'lineNumber') ? $stackframe->lineNumber : "-";
+                $function = property_exists($stackframe, 'functionName') ? $stackframe->functionName : "-";
+                $columnNumber = property_exists($stackframe, 'columnNumber') ? $stackframe->columnNumber : "-";
+                $stacktrace .= "\n$fileName : $function : $lineNumber : $columnNumber";
+            }
+            $context['stacktrace'] = $stacktrace;
```

Comment:
I think it would be nice to add file, line, function and column as separate context keys as well. This allows e.g. searching for these symbols on advanced log processors.